package.json

```diff
@@ -11,9 +11,9 @@
     "build": "NODE_ENV=production npm run build:css && ./build",
     "build-dev": "npm run build:css && SKYNET_MYSKY_ENV=dev ./build",
     "build:css": "postcss assets/style.compile.css -o assets/style.css",
-    "deploy": "npm run build && ts-node deploy.ts",
-    "deploy-dev": "npm run build-dev && DEV=1 ts-node deploy.ts",
-    "deploy-alpha": "npm run build && ALPHA=1 ts-node deploy.ts",
+    "deploy": "npm run build && RESOLVER_DATAKEY=skynet-mysky ts-node deploy.ts",
+    "deploy-dev": "npm run build-dev && RESOLVER_DATAKEY=skynet-mysky-dev ts-node deploy.ts",
+    "deploy-alpha": "npm run build && RESOLVER_DATAKEY=sandbridge ts-node deploy.ts",
```

Comment:
So the idea is to have a deploy script that can be drag-and-dropped into any project, and easily configured by either changing the constants or setting env vars. I do like setting the data key outside of the script instead of having special logic inside it that makes it not reusable.